### ISSUE TITLE
Deletion of default_cert  does not takes effect until force updates thread is executed.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#8578]
Rancher server version - Build from master
LB image - prachidamle/lb-service-haproxy:v0.1

Steps to reproduce the problem:
Create LB services with both certs and default_cert directories .
```
      io.rancher.lb_service.cert_dir: /certs/mycerts
      io.rancher.lb_service.default_cert_dir: /certs/default.com
```
Once LB is running , make sure that we are able to access LB using certs under the default_cert directory.

Delete the directory under ```/certs/default.com``` where default certificates exists.
This change does not get picked up when the next thread for polling changes gets executed,

haproxy continues to be programed with the deleted default cert.

Changes take effect within the next 5 minutes when the "force updates thread" gets run.

```
4/21/2017 11:27:07 AMtime="2017-04-21T18:27:07Z" level=info msg="KUBERNETES_URL is not set, skipping init of kubernetes controller"
4/21/2017 11:27:07 AMtime="2017-04-21T18:27:07Z" level=info msg="Starting Rancher LB service"
4/21/2017 11:27:07 AMtime="2017-04-21T18:27:07Z" level=info msg="LB controller: rancher"
4/21/2017 11:27:07 AMtime="2017-04-21T18:27:07Z" level=info msg="LB provider: haproxy"
4/21/2017 11:27:07 AMtime="2017-04-21T18:27:07Z" level=info msg="starting rancher controller"
4/21/2017 11:27:07 AMtime="2017-04-21T18:27:07Z" level=info msg="Healthcheck handler is listening on :10241"
4/21/2017 11:27:08 AMtime="2017-04-21T18:27:08Z" level=info msg=" -- starting haproxy\n * Starting haproxy haproxy\n   ...done.\n"
4/21/2017 11:27:08 AMtime="2017-04-21T18:27:08Z" level=info msg="LookForCertUpdates: Found an update in cert dir /certs/mycerts, updating the cache"
4/21/2017 11:27:09 AMtime="2017-04-21T18:27:09Z" level=info msg=" -- reloading haproxy config with the new config changes\n * Reloading haproxy haproxy\n[WARNING] 110/182709 (51) : config : 'option forwardfor' ignored for proxy 'default' as it requires HTTP mode.\n[WARNING] 110/182709 (53) : config : 'option forwardfor' ignored for proxy 'default' as it requires HTTP mode.\n   ...done.\n"
4/21/2017 11:32:18 AMtime="2017-04-21T18:32:18Z" level=info msg="LookForCertUpdates: Executing force update as certs in cache have not been updated in: 300 seconds"
4/21/2017 11:32:19 AMtime="2017-04-21T18:32:19Z" level=info msg="LookForCertUpdates: Force Update triggered, updating the cache from cert dir /certs/mycerts"
4/21/2017 11:32:19 AMtime="2017-04-21T18:32:19Z" level=info msg=" -- reloading haproxy config with the new config changes\n * Reloading haproxy haproxy\n[WARNING] 110/183219 (201) : config : 'option forwardfor' ignored for proxy 'default' as it requires HTTP mode.\n[WARNING] 110/183219 (203) : config : 'option forwardfor' ignored for proxy 'default' as it requires HTTP mode.\n   ...done.\n”

```

**Comment by deniseschannon** [comment]
Available in `lb-service-haproxy:v0.7.1`

**Comment by sangeethah** [comment]
Tested with rancher/lb-service-haproxy:v0.7.1 using rancher-server version - v1.6.0-rc1.

This issue is not seen anymore. Deletion of default cert takes effect immediately.
